drivers/sub8_sonar/sub8_sonar/sonar_driver_least_squares.py

```diff
@@ -0,0 +1,242 @@
+#!/usr/bin/env python
```

Comment:
I think it would be good to refactor the two sonar drivers so that they both exist in the same file and the service call contains a string that allows the user to pick which method they want to use.